app/src/main/java/com/example/encoratask/model/Character.kt

```diff
@@ -0,0 +1,46 @@
+package com.example.encoratask.model
+
+import org.json.JSONObject
+import java.io.Serializable
+
+/**
+ * Created by Erick Sanchez
+ * Revision 1 - 09/06/21
+ */
+class Character:Serializable {
+
+    var id:Int = 0
+    var name:String = ""
+    var status = ""
+    var species = ""
+    var gender = ""
+    var image:String = ""
+
+    constructor(obj:JSONObject) {
```

Comment:
I would send this constructor to an extension method. For example

fun Character.fromJSONObject(obj:JSONObject) and call it freely in a Repository class

Reply:
Good idea, but if you wanna use this in Java? it gonna be no friendly

Reply:
The extensions written in Kotlin can also be used in Java with no problem. It only changes quite a little bit
In Java:
`CharacterKT.fromJSONObject(obj)`

Reply:
Ok bro, I got it, good catch, thanks bro